src/main/external-resources/UMS.conf

```diff
@@ -1306,12 +1312,30 @@ web_flash =
 # Default: false
 web_subtitles_transcoded =
 
-# Other web settings
-# ------------------
-# Not documented
+# web videos MIME types spoofing
```

Comment:
All the undocumented are non-GUI, but as the old ones wasn't marked as it, i just did the same.